presto-product-tests/src/main/java/io/prestosql/tests/jdbc/BaseLdapJdbcTest.java

```diff
@@ -82,13 +82,19 @@ public Requirement getRequirements(Configuration configuration)
     }
 
     protected void expectQueryToFail(String user, String password, String message)
+    {
+        assertEquals(expectQueryToFailAndReturnExceptionMessage(user, password), message);
+    }
+
+    protected String expectQueryToFailAndReturnExceptionMessage(String user, String password)
     {
         try {
             executeLdapQuery(NATION_SELECT_ALL_QUERY, user, password);
             fail();
+            return null;
```

Comment:
`throw new AssertionError("Query failure expected");` instead of `fail()`, so you no longer need `return null;`